app/migrations/versions/2591b8be0eb8_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: 2591b8be0eb8
+Revises: 5f4b4c9f586d
+Create Date: 2020-01-26 12:28:39.121376
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '2591b8be0eb8'
+down_revision = '5f4b4c9f586d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('user', sa.Column('is_ussd_self_sign_up', sa.Boolean(), nullable=True))
```

Comment:
Would like to hear what @enjeyw or @tristanhcole think, but I think it would be good if instead of a `is_ussd_self_sign_up` boolean here, we used an `Enum` (or maybe a `String`) called something like `signup_method`. That way we can scale this solution and track things like field signups, or future mechanisms like mobile app signups, web signups, or anything else.

Reply:
I think an enum would probably make a lot of sense here, especially since `is_self_sign_up` is already a field.